views/js/controller/runner/runner.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2018 Open Assessment Technologies SA ;
+ */
+
+/**
+ * Test runner entry point
+ *
+ * @author Bertrand Chevrier <bertrand@taotesting.com>
+ */
+define([
+    'lodash',
+    'taoTests/runner/runnerComponent',
+], function(_, runnerComponentFactory) {
+    'use strict';
+
+    var runnerController = {
+        start : function start(config){
+
+            console.log(config);
```

Comment:
I guess this controller was added for debugging purpose. Is it useful to keep it?

Reply:
good catch I'll remove it